packages/config/config/devices/0x001d/zw4sf.json

```diff
@@ -0,0 +1,112 @@
+// Leviton ZW4SF
+// 4 Speed Fan Controller
+{
+	"manufacturer": "Leviton",
+	"manufacturerId": "0x001d",
+	"label": "ZW4SF",
+	"description": "Decora Smart 4 Speed Fan Controller",
+	"devices": [
+		{
+			"productType": "0x0038",
+			"productId": "0x0002",
+			"zwaveAllianceId": 3832
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": {
+		"3": {
+			"label": "Min Level",
+			"description": "Minimum Fan Speed level allowed",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 10,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"label": "Max Level",
+			"description": "Maximum Fan Speed level allowed",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 99,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5": {
+			"label": "Preset Level",
+			"description": "Preset Level for On",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"6": {
+			"label": "LED Dim Level Indicator Timeout",
+			"description": "The length of time in seconds that the level indicators should stay illuminated after the dimming level is changed.  A value of 0 means the indicator is always off, and a value of 255 means the indicators are always on.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 3,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Level Indicators Always Off",
+					"value": 0
+				},
+				{
+					"label": "Level Indicators Always On",
+					"value": 255
+				}
+			]
+		},
+		"7": {
+			"label": "Locator LED Status",
+			"description": "Change the mode of the LED to Status Mode where the LED is illuminated when the load is On, Locator Mode where the LED is illuminated when the Load is Off, or turns off the LED completely for any other setting values.",
```

Comment:
```suggestion
			"description": "Status Mode: The LED is illuminated when the load is On. Locator Mode: The LED is illuminated when the Load is Off.",
```

Reply:
I suggest losing the desciption altogether. Change this to LED Indicator for the label. The options can be made more descriptive below.

Reply:
I accepted the edits to change the language. Perhaps that's a better route so it's consistent with other params that do include a description? Happy to change it if making the options more descriptive is cleaner!

Reply:
Yes, please do it my way so it matches what we're doing elsewhere.